actions/workflows/delivery_runfolder_workflow.yaml

```diff
@@ -0,0 +1,96 @@
+version: "2.0" # mistral version
+name: arteria-packs.delivery_runfolder_workflow
+description: Deliver data from a runfolder to a SNIC delivery project
+
+workflows:
+    main:
+        type: direct
+        input:
+          - runfolder_name
+          - projects_pi_email_file
+          # TODO Later we want to make sure that we can restrict which projects should be delivered...
+          #- restrict_to_projects
+
+        # FLOW:
+        # [x] Check which projects are available for runfolder
+        # [x] Get emails for PIs for these projects
+        # [x] Get PI id for PIs in SUPr
+        # [x] Stage projects
+        # [x] Create delivery projects for PIs
+        # [] Run delivery
+
+        tasks:
+            note_workflow_version:
+              action: core.local
+              input:
+                cmd: git rev-parse HEAD
+                cwd: /opt/stackstorm/packs/arteria-packs/
+              on-success:
+                  - get_config
+
+            get_config:
+              action: arteria-packs.get_pack_config
+              publish:
+                supr_api_user: <% task(get_config).result.result.supr_api_user %>
+                supr_api_key: <% task(get_config).result.result.supr_api_key %>
+                supr_api_url: <% task(get_config).result.result.supr_api_url %>
+                supr_api_url: <% task(get_config).result.result.supr_api_url %>
+                irma_api_key: <% task(get_config).result.result.irma_api_key %>
+                delivery_service_url: <% task(get_config).result.result.delivery_service_url %>
+              on-success:
+                 - projects_on_runfolder
+
+            projects_on_runfolder:
+              action: core.http
+              input:
+                url: "<% $.delivery_service_url %>/api/1.0/runfolders/<% $.runfolder_name %>/projects"
```

Comment:
I think it's better to keep it here, since we might want to have multiple versions of the api in the future. However, it's of course a matter of taste exactly where to specify it.